packages/hdwallet-walletconnectV2/src/ethereum.ts

```diff
@@ -87,11 +86,9 @@ export async function ethSignMessage(
   args: { data: string | Bytes; fromAddress: string },
```

Comment:
If we remove the string to buffer to hex string dance, we'll have to be 100% sure this comes as a string
```suggestion
  args: { data: string; fromAddress: string },
```
See https://docs.metamask.io/wallet/reference/personal_sign